spec/models/game_spec.rb

```diff
@@ -63,4 +63,12 @@
       end
     end
   end
+
+ describe "#populate_board" do
+    it "should populate board only if both black and white player ids are present" do
+      game = Game.create(white_player_id: 0, black_player_id: 1)
```

Comment:
Where are you getting the values `0` and `1`? These have to be the ids that belong to users that are actually in the database.